sickbeard/scene_exceptions.py

```diff
@@ -273,9 +273,9 @@ def update_scene_exceptions(indexer_id, scene_exceptions, season=-1):
     # A change has been made to the scene exception list. Let's clear the cache, to make this visible
     if indexer_id in exceptionsCache:
         exceptionsCache[indexer_id] = {}
-        exceptionsCache[indexer_id][season] = scene_exceptions
+        exceptionsCache[indexer_id][season] = [se.decode('utf-8', 'ignore') for se in scene_exceptions]
```

Comment:
Yes, I would say the problem is that when you add an exception that contains chars that your locale doesn't support (in your case cp1252) the [join() that takes place in displayShow](https://github.com/pymedusa/SickRage/blob/master/gui/slick/views/displayShow.mako#L186) tries to decode them with the default locale, but fails.